core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -26,6 +28,9 @@ final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRunt
                                   labels: LabelMap = Map.empty,
                                   customEnvironmentVariables: Map[String, String])
 
+final case class BatchNodepoolCreateRequest(numNodepools: NumNodepools,
```

Comment:
Moved this from `leonardoServiceContractModel.scala`, where I think it didn't belong

Reply:
maybe move to https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala?

Reply:
That's where it is now :)

Reply:
oh..was reading the path wrong